integration-test/src/main/java/com/sequenceiq/it/cloudbreak/RegionTests.java

```diff
@@ -160,7 +161,7 @@ public void testListAvailabilityZonesForDefaultRegion() throws Exception {
                             .get(cloudProvider.region());
 
                     if (cloudProvider.getPlatform().equalsIgnoreCase("AZURE")) {
```

Comment:
The conditional part could be another test, probably platform dependent. In general a test should be as straightforward as possible, any condition in a test case means another test case; common parts could be in BeforeTest or other method.